hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestMiniOzoneCluster.java

```diff
@@ -132,6 +134,13 @@ public void testDatanodeIDPersistent() throws Exception {
     File validIdsFile = new File(WRITE_TMP, "valid-values.id");
     validIdsFile.delete();
     ContainerUtils.writeDatanodeDetailsTo(id1, validIdsFile);
+    // Validate using yaml parser
+    Yaml yaml = new Yaml();
+    try {
+      yaml.load(new FileReader(validIdsFile));
+    } catch (Exception e) {
```

Comment:
Why not just let the exception bubble up?

Reply:
Caller expects IOException and yaml parser can throw its own Checked types.